nss-evening-cohort-14/jurrasic-world-dashboard-jurrasic-world-dashboard
# Chaos Monkey, styling, README

## Netlify: 
jurassic-world-dashboard.netlify.app

## Description
- We created a toast 
- We added functionality to show the toast after a certain amount of time 
- The toast displays a randomized message 
- We created a button to start and stop the toast
- We added a monkey animation which flies across the screen when the monkey is released 
- We added styling to overall page, cards and chaos monkey 
- Updated README with Loom video 

## Related Issue
#64 
#65 
#63 
#59 
#20 
#70 

## Motivation and Context
- Chaos Monkey was part requirements. 
- Styling and animation enhances user experience. 
- README is required as well. 

## How Can This Be Tested?
- Check out Netlify link 
- Fetch local branch and serve

## Types of changes
